Posit Infrastructure/New Posit Environment Quick Start Guide.md

```diff
@@ -0,0 +1,256 @@
+
+- Once you click the "Start Session" button, a further dialog will appear in the bottom-right of your browser window giving you updates on the progress of starting the session:
+
+![Starting Session Dialog](https://github.com/user-attachments/assets/2966723d-9b0a-42e2-99cb-9b6a8c1aa6e7)
+
+
+**Important Note: normally sessions will take just a few seconds to start, but if the Kubernetes cluster needs to scale up to accommodate your session, it can take up to 10 minutes for your session to start.  This is normal and expected behaviour.**
+
+- The session will be listed on the Posit Workbench homepage as "PENDING" during this process:
+
+![RStudio Pro Session Pending](https://github.com/user-attachments/assets/0713f025-c984-4a46-9c48-bbbf2fb1ea35)
+
+- If the "Join session when ready" checkbox was ticked in the "New Session" dialog window, the session will automatically load in your browser.  Alternatively, once the session shows as "ACTIVE" on the Posit Workbench homepage, you can click the session to open it:
+
+![RStudio Pro Session Active](https://github.com/user-attachments/assets/e7e81280-e71b-4d5e-9742-4ca2650bb9b3)
+
+
+## 5. Configuring your RStudio Pro environment
+
+### Global Options
+
+- Once your RStudio Pro session has started, please ensure that you configure your settings as follows:
+  - Access the RStudio Global Options menu by going to _Tools > Global Options..._
+  - In the General menu, be sure that the settings match those in the screenshot below:
+
+![Global Options](https://github.com/user-attachments/assets/c1441dba-b6af-4bec-b017-cc371bd14091)
+
+  - _Optional, but recommended:_ On the Appearance menu, change the editor font to "Cascadia Code".  This font has support for ligatures that map multiple characters to single glyphs e.g. rendering "<-" as a single arrow glyph.
+
+![Cascadia Code](https://github.com/user-attachments/assets/591ad6c5-4fad-4c25-9142-4ae0df288428)
+
+### Installing Packages
+
+- You will need to install R packages that you need.  These will not transfer over from the old Posit environment.  In the old Posit environment, you can run the following R code to write out a CSV file listing the packages you had installed.  This might be helpful for re-installing the packages you need:
+
+```r
+# Function to check and install rstudioapi if necessary
+install_rstudioapi <- function() {
+  if (!requireNamespace("rstudioapi", quietly = TRUE)) {
+    cat("The 'rstudioapi' package is not installed. Installing it now...\n")
+    install.packages("rstudioapi", quiet = TRUE)
+    if (!requireNamespace("rstudioapi", quietly = TRUE)) {
+      stop("Failed to install 'rstudioapi'. Please install it manually.")
+    }
+    cat("'rstudioapi' package has been successfully installed.\n")
+  }
+}
+
+# Install rstudioapi if not present
+install_rstudioapi()
```

Comment:
I'm certain this first bit isn't required, as `{rstudioapi}` will be installed by default on any RStudio instance.
```suggestion
```

Also, it would be useful to have instructions on how to use the saved CSV file to restore/install packages.

Reply:
It's intended that this R code is run on the old Posit environment.  `{rstudioapi}` is not installed by default on the old Posit environment.  Users need to install themselves first.  I'll leave this bit in.

Reply:
Just to say, we will not have time / resource to implement and test a solution for taking the exported CSV file with the list of R packages, and automate the installation of this list in the new Posit Workbench environment.